src/index.html

```diff
@@ -3,10 +3,457 @@
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__add-info">
+        Product code: 195434
+      </p>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+        </div>
+
+        <p class="card__review">
+          Review:5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-text">
+          Price:
+        </p>
+
+        <p class="card__price-amount">
+          $2,199
+        </p>
+      </div>
+
+      <a
+        class="card__link"
+        href="#buy"
+        data-qa="hover"
+      >
+      BUY
+      </a>
+    </div>
+    <div class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__add-info">
+        Product code: 195434
+      </p>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+        </div>
+
+        <p class="card__review">
+          Review:5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-text">
+          Price:
+        </p>
+
+        <p class="card__price-amount">
+          $2,199
+        </p>
+      </div>
+
+      <a
+        class="card__link"
+        href="#buy"
+        data-qa="hover"
+      >
+      BUY
+      </a>
+    </div>
+    <div class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__add-info">
+        Product code: 195434
+      </p>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+
+          <div class="stars__item"></div>
+        </div>
+
+        <p class="card__review">
+          Review:5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-text">
+          Price:
+        </p>
+
+        <p class="card__price-amount">
+          $2,199
+        </p>
+      </div>
+
+      <a
+        class="card__link"
+        href="#buy"
+        data-qa="hover"
+      >
+      BUY
+      </a>
+    </div>
+  </main>
+
   <body>
```

Comment:
The <body> tag should be placed before the <header> tag, not after the <main> tag. Move the <body> tag to line 10.